litellm/integrations/langfuse.py

```diff
@@ -94,9 +115,7 @@ def log_event(
 
             litellm_params = kwargs.get("litellm_params", {})
             litellm_call_id = kwargs.get("litellm_call_id", None)
-            metadata = (
-                litellm_params.get("metadata", {}) or {}
-            )  # if litellm_params['metadata'] == None
+            metadata = self.add_metadata_from_header(litellm_params)
```

Comment:
this is confusing. Can you keep the line. Then after that line you can set `metadata = self.add_metadata_from_header(metadata)` 

```python
metadata = (
                litellm_params.get("metadata", {}) or {}
            )  
            
 ```